guides/common/modules/proc_running-installer.adoc

```diff
@@ -0,0 +1,17 @@
+[id="running-installer_{context}"]
+= Running the Installer
+
+The installation run is non-interactive, but the configuration can be customized by supplying any of the options listed in `foreman-installer --help`, or by running `foreman-installer -i` for interactive mode.
+More examples are given in the Installation Options section.
```

Comment:
I assume "Installation Options" refers to stdout from "foreman-installer --help". Does it need some emphasis then?